src/test/encoding/import-generated.sh

```diff
@@ -4,27 +4,33 @@ archive=$1
 
 [ -d "$archive" ] || echo "usage: $0 <archive>"
 
-ver=`./ceph-dencoder version`
+if [ -e ceph-dencoder ]; then
```

Comment:
this assumes that user might want run this script in the directory where the `ceph-dencoder` is placed. but i think in realworld, this does not really happen. as user either install `ceph-dencoder` into `/usr/bin` or want run it in `build/bin`. the latter is taken care of by the `elif` branch.

if we really want to appease the first case, you might want to use `which` instead of `-e`. but i doubt it's necessary. as developer always use the executables in `build/bin` without actually installing the artifacts.

Reply:
1. With the above change, it could run successfully as said in doc/dev/corpus.rst
2. The added path environment is in the end of PATH environment variable, so it won't affect /bin/ceph-dencoder if it really exist on the system.
3. "which" seems not needed since the script will abort execution ceph-denconder isn't found.